v3.0/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -133,20 +133,14 @@ spec:
             # Cluster type to identify the deployment type
             - name: CLUSTER_TYPE
               value: "k8s,bgp"
+            # Disable file logging so `kubectl logs` works.
+            - name: CALICO_DISABLE_FILE_LOGGING
+              value: "true"
             # Set noderef for node controller.
             - name: CALICO_K8S_NODE_REF
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            # Enable Felix IPInIPEnabled, pending code changes
-            - name: FELIX_IPINIPENABLED
-              value: "true"
-            # Disable node ip check, pending code changes
-            - name: DISABLE_NODE_IP_CHECK
-              value: "true"
-            # Disable file logging so `kubectl logs` works.
-            - name: CALICO_DISABLE_FILE_LOGGING
```

Comment:
Why is this being removed? It exists in master
https://github.com/projectcalico/calico/blob/master/master/getting-started/kubernetes/installation/hosted/calico.yaml#L137

Reply:
Oh, I see it's getting moved above. Can we just leave it where it is?

Reply:
right now, master == v3.0 which I think is useful. your call though. let me know what you want.